### ISSUE TITLE
Protocol upgrade tracking

### ISSUE THREAD

**Issue by ilblackdragon** [near/near-explorer#444]
We have protocol upgrades that are defined by version of the protocol that validators run.

It's very important roles in the ecosystem to see this information:
 - developers - they want to know when the features will be launched
 - node runners (both validators and non validators) to understand that they need to update

Ideally we should show:
 -  current protocol version
 - % of stake that run next protocol version
 - if the next protocol version already hit threshold (80%) have a warning about when next protocol version will be live (2 epochs after).

**Comment by frol** [comment]
For the context, `status` RPC returns `protocol_version` and `latest_protocol_version`, but to collect the information, we would need to call `status` RPC on every producing (a.k.a. validating) node, yet we don't have this opportunity. Thus, we need to extend nearcore Telemetry first.

**Comment by bowenwang1996** [comment]
> For the context, status RPC returns protocol_version and latest_protocol_version, but to collect the information, we would need to call status RPC on every producing (a.k.a. validating) node, yet we don't have this opportunity. Thus, we need to extend nearcore Telemetry first.

I don't think it should be done this way because telemetry is optional. We need to change nearcore to expose the last voted protocol version.

**Comment by frol** [comment]
@bowenwang1996 exposed through `validators` API or how exactly?

**Comment by bowenwang1996** [comment]
> @bowenwang1996 exposed through validators API or how exactly?

Yes. This requires changing some internals as well
